library/node

```diff
@@ -3,114 +3,114 @@
 Maintainers: The Node.js Docker Team <https://github.com/nodejs/docker-node> (@nodejs)
 GitRepo: https://github.com/nodejs/docker-node.git
 
-Tags: 15.2.1-alpine3.10, 15.2-alpine3.10, 15-alpine3.10, alpine3.10, current-alpine3.10
+Tags: 15.4.0-alpine3.10, 15.4-alpine3.10, 15-alpine3.10, alpine3.10, current-alpine3.10
```

Comment:
This skips 15.3.0 - do we care enough to land that first?

/cc @LaurentGoderre @PeterDaveHello @SimenB @Starefossen @nschonni @ttshivers

Reply:
probably... let me open a PR with that

Reply:
It already landed https://github.com/docker-library/official-images/commit/bc9ab764e1145fcda6f8266aaeb6bd92f1d61253, I think your branch is behind

Reply:
wait, I just forgot to pull 😅